### ISSUE TITLE
Other log files significantly grow in size

### ISSUE THREAD

**Issue by jbob182** [jbob182/pia-openvpn-wrapper#1]
While the `openvpn.log` file is now regulated, the other log files still are capable of being quite large in size.

**Comment by p1r473** [comment]
Will you add support for the other logs then?

**Comment by jbob182** [comment]
Yes, I am currently waiting for a user to report if adding the `nolog` file suppressed the other logs.

**Comment by p1r473** [comment]
It doesn't seem to suppress them, only delete them when program is exited
